README.md

```diff
@@ -8,3 +8,45 @@ SPDX-License-Identifier: Apache-2.0
 
 This repository contains the `.proto` files used by the Astarte message hub as well as by any
 client that wishes to connect itself to the message hub.
+
+## Code generation
+
+This project contains a Makefile that allows you to generate code for all supported languages.
+Currently, the language supported are:
```

Comment:
I suggest changing the last line into "Currently, the following languages are supported:" or "Currently, the supported languages are:"